app/views/site/contribute.erb

```diff
@@ -0,0 +1,168 @@
+<div class="container">
+  <section class="contributing-intro jumbotron">
+    <h1>Contribute to Exercism</h1>
+
+    <p class="lead">
+      <strong>So you want to contribute to exercism?</strong><br />
+      That is great news! We welcome all sorts of contributions and will do everything we can to help you get started.
+    </p>
+
+    <p>
+      <strong>Exercism is a project comprised of several repositories and these can be divided basically in:</strong>
+
+      <ol>
+        <li><strong>The website</strong>: Browse and review solutions, and have conversations.</li>
```

Comment:
_Holds the code that people submit. Browse and review solutions, and have conversations._